cns-hook-util

```diff
@@ -105,17 +105,52 @@ function mdata_delete {
 	$cmd "$@"
 }
 
+function merge_token {
+	local domain="${1}" tokenval="${2}"
+	local existing_token
+
+	local zname=$(zonename)
+	if [ "$zname" != "global" ]; then
+		existing_token=$(mdata-get triton.cns.acme-challenge)
```

Comment:
I think we should make a function for calling mdata-get (like the existing `function mdata_put` further down) which deals with finding it in `/native` if we're on LX?

Otherwise you have to have PATH set to include /native to run this. (I think we should do the same for the `zonename` issue you discovered, too -- make it into a bash function that goes and finds it in `/native` if we're in LX)